### DISCUSSION TITLE
Non-global shortcuts

### DISCUSSION THREAD

**Discussion by nanodeath** [tauri-apps/tauri#5242] [Q&A] [answered]
Hi all,

Is there a Tauri-specific way to handle regular (application-wide, or at least window-wide) hotkey combinations? I'm trying to make Ctrl+S work, and discovered the hard way that global shortcuts are indeed global as in _system-wide_, not _application-wide_. I could probably do it in vanilla JS but it seems like there are some cross-OS concerns there that I'd rather avoid if possible. Might have to do it that way though.

**Comment by FabianLars** [comment, marked as answer]
It currently is indeed only possible in js or via some rust crate, but we're considering implementing (aka. we want to, but don't have the capacity for it - also it may have to wait for the winit merge).

If you'd like you can open a feature request so that it's publicly tracked

**Reply by nanodeath** [reply]
Cool, I might do that, thanks. Originally I was hoping that just by making `Ctrl-S` an accelerator for a menu item that would Just Work, but maybe that's something else entirely.
